### PR TITLE
card component

### PR DESCRIPTION
**Pull request by Rockyraven**
some CSS and HTML change
more component and button added

### CODE REVIEW

File: component/card/card.css [hunk 1]
```diff
@@ -1,112 +1,108 @@
...
 .buy-button ,.read-more{
     background-color: var(--button-color);
-     height: 26px;
-     width: 95%;
-     text-align: center;
-     border-radius: 5px;
-     margin-left: 5px;
-     color: var(--default-color);
-     padding-top: 5px;
+    height: 26px;
```
Comment (sagargse) [inline component/card/card.css]: you can use rem here

**Review by sagargse — APPROVED** [review]
rest all looks good to merge

File: component/card/card.css (hunk 1 above)
Comment (Rockyraven) [reply component/card/card.css]: I have change in rm

**Review by arjun-computer-geek — APPROVED** [review]

**Review by Developers-codz — APPROVED** [review]
Looks good to me

**Pull request merged**